### PR TITLE
fix 150

### PR DESCRIPTION
**Pull request by brendansudol**
fixes https://github.com/18F/dol-whd-14c/issues/150

### CODE REVIEW

**Comment by mgwalker** [comment]
@EStriegel I'd like your input on this one.  Previously, the question text was one of these two:

> How many workers did the employer pay hourly?
> How many workers did the employer pay on a piece rate basis?

With @brendansudol's edits, the question texts are now:

> How many workers with disabilities were paidan hourly subminimum wage during the most recently completed fiscal quarter?
> How many workers with disabilities were paid on a piece rate basis subminimum wage during the most recently completed fiscal quarter?

The new piece rate question seems a little awkward to me.  What do you think?

**Comment by EStriegel** [comment]
@mgwalker Good catch - the piece rate question should read, "How many workers with disabilities received subminimum wages and were paid piece rates during the most recently completed fiscal quarter?"

Also, not sure if it's just a typo in the comment, but there's a space missing in the hourly question, "paidan hourly"..

**Comment by mgwalker** [comment]
Whoops.  Typo on my part!  😄

**Comment by brendansudol** [comment]
made those updates -- thanks for weighing in @mgwalker and @EStriegel :)

File: DOL.WHD.Section14c.Web/src/modules/components/sectionAdminWageData/adminWageDataPayTypeTemplate.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <answer-field answer="data.numWorkers">
-    How many workers did the employer pay {{ paytype === $parent.$parent.$parent.constants.responses.payType.hourly ? 'hourly' : 'on a piece rate basis'}}?
+    {{ paytype === $parent.$parent.$parent.constants.responses.payType.hourly ? 'How many workers with disabilities were paid an hourly subminimum wage during the most recently completed fiscal quarter?' : 'How many workers with disabilities received subminimum wages and were paid piece rates during the most recently completed fiscal quarter?' }}
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/components/sectionAdminWageData/adminWageDataPayTypeTemplate.html]: To be honest, I like this approach way better than the other.  It's a lot easier to read and reason about.

**Pull request merged**